packages/core/src/families/bitcoin/validation.ts

```diff
@@ -4,4 +4,5 @@ import { schemaFamilies, schemaTransactionCommon } from "../common";
 export const schemaRawBitcoinTransaction = schemaTransactionCommon.extend({
   family: z.literal(schemaFamilies.enum.bitcoin),
   feePerByte: z.string().optional(),
+  opReturnData: z.string().max(80).optional(),
```

Comment:
80 bytes in a buffer != 80 characters in a utf8 string. Is this intended ?

We tend to transport raw data has hex strings. When it is the case, please include "hex" in the field's name so we know we are dealing with serialized data :)  We would also need to calculate the appropriate max string size for hex data.

Reply:
From your Slack response

```
1 hex character = 4 bits
2 hex characters = 8 bits = 1 byte
80 bytes = 160 hex characters
```